packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -0,0 +1,512 @@
+                layoutMode={DetailsListLayoutMode.justified}
+                items={properties}
+                columns={isEnum ? this._enumColumns : this._defaultColumns}
+                onRenderRow={this._onRenderRow}
+              />
+            )}
+      </Stack>
+    );
+  }
+
+  private renderCell = (text: string, deprecated: boolean = false) => {
+    // When the text is passed to this function, it has had newline characters removed,
+    // so the regex will match backtick sequences that span multiple lines.
+    let regexResult: RegExpExecArray | null;
+    const codeBlocks: { index: number; text: string }[] = [];
+    while ((regexResult = backticksRegex.exec(text)) !== null) {
+      codeBlocks.push({
+        index: regexResult.index,
+        text: regexResult[0]
+      });
+    }
+
+    if (codeBlocks.length === 0) {
+      return referencesTableCell(text, deprecated);
+    }
+
+    const textElements = this._extractCodeBlocks(text, codeBlocks);
+
+    return referencesTableCell(textElements, deprecated);
+  };
+
+  private renderCellType = (typeTokens: ILinkToken[]) => {
+    return this._parseILinkTokens(false, typeTokens);
+  };
+
+  private createRenderCellEnum = (propertyName: keyof IApiEnumProperty) => (item: IApiEnumProperty) => this.renderCell(item[propertyName]);
+
+  private createRenderCellInterface = (propertyName: 'name' | 'description' | 'defaultValue') => (item: IApiInterfaceProperty) =>
+    this.renderCell(item[propertyName], propertyName === 'description' && item.deprecated);
+
+  private createRenderCellType = (propertyName: 'typeTokens') => (item: IApiInterfaceProperty) => this.renderCellType(item[propertyName]);
+
+  private createRenderCellSignature = (propertyName: 'typeTokens') => (item: IMethod) => this.renderCellType(item[propertyName]);
+
+  /**
+   * Loops through text and places code blocks in code elements
+   */
+  private _extractCodeBlocks(text: string, codeBlocks: { index: number; text: string }[]): JSX.Element[] {
+    const textElements: JSX.Element[] = [];
+
+    let codeIndex = 0;
+    let textIndex = 0;
+    while (textIndex < text.length && codeIndex < codeBlocks.length) {
+      const codeBlock = codeBlocks[codeIndex];
+      if (textIndex < codeBlock.index) {
+        const str = text.substring(textIndex, codeBlock.index);
+        textElements.push(<span key={textIndex}>{str}</span>);
+        textIndex += str.length;
+      } else {
+        textElements.push(<code key={textIndex}>{codeBlock.text.substring(1, codeBlock.text.length - 1)}</code>);
+        codeIndex++;
+        textIndex += codeBlock.text.length;
+      }
+    }
+    if (textIndex < text.length) {
+      textElements.push(<span key={textIndex}>{text.substring(textIndex, text.length)}</span>);
+    }
+
+    return textElements;
+  }
+
+  private _renderClass(): JSX.Element | undefined {
+    const { properties, methods } = this.state;
+
+    return methods ? (
+      <Stack gap={MEDIUM_GAP_SIZE}>
```

Comment:
Not sure nested stacks are the best way to go here, since there are only two elements per stack, and stacks use flexbox (which does have a higher perf cost). Might be better to just use margin after the text or before the list instead. (Also, would be nice if the "Members" part could be de-duplicated.)

Reply:
@khmakoto Do you know how much of a performance concern it is to have nested stacks like this? (I vaguely remember it being a concern, but I don't remember the details)

Reply:
Using flexbox in and of itself introduces performance issues. I don't have a specific metric on how much its use impacts performance. I would say that for these two item scenarios it might be better to just have a margin on the `Text`, especially if the `Stacks` are only being used to add a gap between the items. However, if the Stacks are being used for other layout purposes, then I would say using nested `Stacks` might still be a good call.

Reply:
Let's revisit this with snapshot captures / measurements given the scenarios we need to support.